src/PostgREST/CLI.hs

```diff
@@ -57,7 +57,7 @@ dumpSchema appState = do
       (transaction SQL.ReadCommitted SQL.Read $ querySchemaCache conf)
   case result of
     Left e -> do
-      hPutStrLn stderr $ "An error ocurred when loading the schema cache:\n" <> show e
+      hPutStrLn stderr $ "Failed to load the schema cache:\n" <> show e
```

Comment:
How about using `observer $ SchemaCacheErrorObs e` here? Just to have all the error messages in one place.